xbmc/windowing/gbm/DRMAtomic.cpp

```diff
@@ -22,6 +24,35 @@
 #include <unistd.h>
 
 using namespace KODI::WINDOWING::GBM;
+const std::string SETTING_VIDEOSCREEN_HW_SCALING_FILTER = "videoscreen.hwscalingfilter";
+bool CDRMAtomic::SetScalingFilter(struct drm_object* object, const char* name, const char* type)
+{
+  uint32_t w_scale_factor;
+  uint32_t h_scale_factor;
+  for (uint32_t i = 0; i < object->props->count_props; i++)
+  {
+    if (!strcmp(object->props_info[i]->name, name))
+    {
+      for (uint32_t j = 0; j < object->props_info[i]->count_enums; j++)
+      {
+        if (!strcmp(object->props_info[i]->enums[j].name, type))
+        {
+          if (AddProperty(object, name, m_gui_plane->props_info[i]->enums[j].value))
+          {
+            CLog::Log(LOGDEBUG, "CDRMAtomic::%s, m_width[%d] x m_height[%d]", __FUNCTION__, m_width,
```

Comment:
please use the new logging format (replace `%s` and `%d` with just `{}`)

Reply:
Sure, I will use the new logging format.